app/models/member.rb

```diff
@@ -16,6 +16,8 @@ class Member < ActiveRecord::Base
   belongs_to :user
   belongs_to :organization
 
+  has_many :invitations, foreign_key: :invited_by_id
```

Comment:
@Gataniel rename this association to something like `created_invitations` to differ from `accepted_invitations`